src/Redux/Actions/users.js

```diff
@@ -0,0 +1 @@
+// Create actions and reducers
```

Comment:
- [ ]   This file looks like is not being used.  Kindly delete it if you are not using it.